readthedocs/projects/tasks.py

```diff
@@ -370,7 +377,7 @@ def run(self, version_pk):  # pylint: disable=arguments-differ
                 version=self.version,
                 record=False,
                 update_on_success=False,
-                environment=self.get_rtd_env_vars(),
+                environment=self.get_vcs_env_vars(),
```

Comment:
Is this logic not changing? We should continue to pass the `rtd_env_vars` here, no?

Reply:
I'd also like to see `get_env_vars` renamed to `get_build_env_vars` if we're doing this refactor, to keep things clear.